pkg/client/injection/reconciler/networking/v1alpha1/certificate/reconciler.go

```diff
@@ -183,7 +183,7 @@ func (r *reconcilerImpl) Reconcile(ctx context.Context, key string) error {
 	logger := logging.FromContext(ctx)
 
 	// Initialize the reconciler state. This will convert the namespace/name
-	// string into a distinct namespace and name, determin if this instance of
+	// string into a distinct namespace and name, determine if this instance of
```

Comment:
It's possible this comes from somewhere. Let me know and I'll try to send a patch to it at some point.

Reply:
https://github.com/knative/pkg/pull/1797